generators/app/GithubValidator.js

```diff
@@ -12,7 +9,13 @@ const validateRepoUrl = (inputUrl) => {
         throw new Error("You must enter a url with the format of `https://{githubBaseUrl}/{owner}/{repo}.`");
     }
 
-    return new GithubUrl(url.origin, owner, repo);
```

Comment:
I had no idea which was better. I know that loose typed objects are possible, but for cleaner code, I wondered if js devs create classes like `GithubUrl` so there's no guessing as to what it contains. Otherwise we have to dig into the code every time to figure it out, right?